inbm/dispatcher-agent/tests/unit/schedule/test_sqlite_manager.py

```diff
@@ -0,0 +1,100 @@
+from unittest import TestCase
+
+from dispatcher.schedule.sqlite_manager import SqliteManager
+from dispatcher.schedule.schedules import SingleSchedule, RepeatedSchedule
+from dispatcher.dispatcher_exception import DispatcherException
+
+class TestSqliteManager(TestCase):
+    
+    def setUp(self) -> None:
+        self.db = SqliteManager(":memory:")
+     
+    def test_raise_exception_when_create_single_schedule_with_invalid_start_time(self):
+        ss1 = SingleSchedule(request_id="REQ123", 
+                            start_time=None,
+                            end_time="2024-01-02T00:00:00", 
+                            manifests=["MANIFEST1", "MANIFEST2"])
+        with self.assertRaisesRegex(DispatcherException, 
+                                    'Error adding single schedule to database: NOT NULL constraint failed: single_schedule.start_time'):
```

Comment:
<h3>$\color{#FFD700}Coding Style$</h3>

Ensure consistent use of single quotes or double quotes for strings throughout the code for consistency.